### ISSUE TITLE
Stipendium profiles

### ISSUE THREAD

**Issue by dominikgeissler** [sbsev/svelte-site#53]
As discussed yesterday, we should add a short overview about the students who have recieved the scholarship with

* An image
* Information about the student and the support (name, year of birth, education, time period of support, financial scope of the support)
* a short quote regarding the support

If there are any more criterion, I will add them to the list above.

**Comment by dominikgeissler** [comment]
@janosh you have already used a method you could recommend in your tikz-project right?

**Comment by janosh** [comment]
Yes, I had suggested a masonry layout. But on second thought, maybe a simple CSS grid like on our blog index page would be better.
